sandbox/public/index.html

```diff
@@ -38,10 +38,10 @@
       alloy("configure", {
         // edgeDomain: "edgegateway.azurewebsites.net",
         // optInEnabled: true,
-        // suppressErrors: true,
+        // errorsEnabled: false,
         propertyId: 9999999,
         imsOrgId: "53A16ACB5CC1D3760A495C99@AdobeOrg",
-        log: true,
+        logEnabled: true,
         // Target VEC appends "mobxEdit" to URL when loading the page.
         // By checking this params we know we are in authoring mode
         authoringMode: document.location.href.indexOf("mboxEdit") !== -1, 
```

Comment:
Should `authoringMode` be `authoringModeEnabled` then? I think for consistency it should.

Reply:
Good catch, I'll add that as well.

Reply:
@Aaronius Added one more commit to address this comment. Thanks!

Reply:
I think you need to update `authoringMode` in index.html still.